GITOID_URI_SPEC.md

```diff
@@ -0,0 +1,44 @@
+Scheme name: gitoid
+
+Status: provisional
+
+Applications/protocols that use this scheme name:
+     See [Section 3.5](https://www.rfc-editor.org/rfc/rfc7595.html#section-3.5).
+Contact:
+     Registering party: Ed Warnicke <eaw@cisco.com>
+
+Scheme Creator: GitBOM
+
+Change controller:
+    Either the registering party or someone who is verified to represent
+    the scheme creator.  See previous answer.
+
+References:
+https://git-scm.com/book/en/v2/Git-Internals-Git-Objects
+
+Scheme syntax: gitoid":"<git object type>":"<hash algorithm>":"<hash value>
+
+Current valid values for <git object type> are 'blob','tree','commit', and 'tag'.
+Current valid values for <hash algorithm> are 'sha1' and 'sha256'
+<hash value> should be expressed as a hexidecimal string in lower case.
+
+Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+
+Scheme semantics:
+
+gitoid stands for Git Object ID.  Git is an object store.  It currently has four types of objects.  A git object is formed by prepending an object header to the contents of the object.  A git object header consists of:
+
+<git object type>" "<size of contents as decimal string>"\0"
+
+The git object id is computed by applying a hash to the git object.  Currently the most common hash is sha1, but there is some effort to transition to sha256 over time.
```

Comment:
s/Currently/Currently,/

Reply:
Also, this line may end up changing... e.g. if we go with the unified line where we have more than one hash. Might be `sha1+sha256` or some similar derivative

Reply:
> s/Currently/Currently,/

I believe I've fixed this.

Reply:
> Also, this line may end up changing... e.g. if we go with the unified line where we have more than one hash. Might be `sha1+sha256` or some similar derivative

Let's leave this be for the moment.